internal/store/configmap_test.go

```diff
@@ -54,6 +41,12 @@ func TestConfigMapStore(t *testing.T) {
 				},
 			},
 			Want: `
+				# HELP kube_configmap_annotations Kubernetes annotations converted to Prometheus labels.
```

Comment:
These are the sortings I was referring to in the description. We first get all the `HELP` and then the `TYPE`, is this a problem for anyone? We still get the correct headers and correct metrics, it's just they are sorted.

Reply:
Not a problem. Prometheus handles this "correctly".